client/src/js/directives/totalsfooter.js

```diff
@@ -38,6 +41,14 @@ function TotalsFooterDirective() {
       // (defaults to 1 column)
       var leadingColumns = scope.leadingColumns || 1;
 
+      // the $rootScope will send a 'nav:toggle' event when the navigation is toggled open/close.
+      // this allows us to update the column when the navigation flexed.
+      // FIXME hardcoded navigation offset
+      scope.$on('nav:toggle', function (e, open) {
+        navigationOffset = (open) ? 260 : 0;
```

Comment:
With this variable being set as it is used it might be hard to track down in the future, or understand the code (I realise  this is probably a temporary solution anyway). 

A way of mitigating this may be to set this value globally using this suggestion https://github.com/IMA-WorldHealth/bhima-2.X/issues/460 . At least then the same value is shared across the application.